docs/_providers/hostingde.md

```diff
@@ -38,3 +40,21 @@ D('example.tld', REG_HOSTINGDE, DnsProvider(DNS_HOSTINGDE),
     A('test', '1.2.3.4')
 );
 {% endhighlight %}
+
+## Customize nameservers
```

Comment:
```suggestion
## Using this provider with HTTP.NET and others

HTTP.NET and other DNS service providers use an API that is compatible with hosting.de.  Using them requires overriding the default nameservers.

### Example HTTP.NET configuration:

``js
var DNS_HTTPNET = NewDnsProvider('http.net', 'HOSTINGDE', {
  default_ns: [
    'ns1.routing.net.',
    'ns2.routing.net.',
    'ns3.routing.net.',
  ],
});
``

Why this works:  hosting.de has the concept of *nameserver sets* but this provider does not implement it.  The `HOSTINGDE` provider **ignores the default nameserver set** defined in your account!  Instead, it uses hosting.de's nameservers (`ns1.hosting.de.`, `ns2.hosting.de.`, and `ns3.hosting.de.`) by default, regardless of your account settings.  The above examples use the `default_ns` meta data to tell the provider what to use instead.
```

As we find other compatible services it will be easy to add them as examples.

NOTE: The 3x backticks are replaced by 2x backticks in the above suggestion due to quoting rules.

Reply:
> Would you also add HTTP.NET to the README.md and docs/provider-list.md files?

Sounds good! I'll happily do so.

> NOTE: The 3x backticks are replaced by 2x backticks in the above suggestion due to quoting rules.

Ah, so a GitHub limitation? Got it :D